src/main/proto/wfa/measurement/api/v2alpha/differential_privacy.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.api.v2alpha;
+
+option java_package = "org.wfanet.measurement.api.v2alpha";
+option java_multiple_files = true;
+option java_outer_classname = "DifferentialPrivacyProto";
+
+// Parameters for differential privacy (DP).
+//
+// For detail, refer to "Dwork, C. and Roth, A., 2014. The algorithmic
+// foundations of differential privacy. Foundations and Trends in Theoretical
+// Computer Science, 9(3-4), pp.211-407."
+message DifferentialPrivacyParams {
```

Comment:
Is this immutable? Is this something that all parties need to agree upon?

Reply:
I think something went wrong in merging -- this definitely shouldn't be part of this PR. This is from a PR that Sanjay already landed.